src/Widgets/LinkWidget/LinkWidgetEditingConfig.js

```diff
@@ -10,4 +10,15 @@ Scrivito.provideEditingConfig("LinkWidget", {
     },
   },
   properties: ["link"],
+  validations: [
+    [
+      "link",
+
+      link => {
+        if (!link) {
+          return { message: "The link should be set.", severity: "warning" };
+        }
+      },
```

Comment:
Let's add a warning, if the links text is not set.

Reply:
This creates a lot of warnings for e.g. internal links. See:

<img width="1440" alt="Bildschirmfoto 2020-01-13 um 12 58 27" src="https://user-images.githubusercontent.com/915585/72254531-97a28100-3604-11ea-98b4-82700e794725.png">

Reply:
Good catch! Also LinkWidgetComponent provides reasonable fallbacks, if no link title is set. So let's leave the validations (empty link) as they are.